README.md

```diff
@@ -1 +1,185 @@
-# AWESOMEBOOK-NAV
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Awesome Books <a name="about-project"></a>
+
+**This is the Awesome Books Project in order to show what we have learned so far at HTML, CSS and JS.**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>Html</li>
+    <li>Css</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **adds a book name and author**
+- **Removes a book from the list**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://citec-47.github.io/AWESOMEBOOK-NAV/)
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Using it Locally
+
+- Clone the project from GitHub `https://github.com/citec-47/AWESOMEBOOK-NAV.git`
+- Run the following commands as listed in your terminal:
+- `npm install`
+- `npm start`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/citec-47/AWESOMEBOOK-NAV.git
+```
+
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd todolist
+  node version 16 or higher
+```
+### Usage
+To run the project, execute the following command:
+
+npm start
+### Testing
+
+ npm run test
+
+### Deployment
+
+Github Pages
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [NdonyiMaurice](https://github.com/citec-47)
+- Twitter: [@NdonyiMaurice](https://twitter.com/Ndonyi4)
+- LinkedIn: [NdonyiMaurice](https://www.linkedin.com/in/ndonyi-maurice-b5b49b22b/)
+
+
+<!-- FUTURE FEATURES -->
+
```

Comment:
You both have done great work on this repository. Everything works properly and the styling is top-notch. You should show off properly 😉. On line 139 here, I suggest you replace ```Author1``` with your name. Also, remember this is a pair programming project and so there should be a second author and I suggest you add his name and details as well. Again, awesome work guys! 💪 and great work on the styling 👍